main.py

```diff
@@ -148,3 +148,4 @@ def register_user(user: CreateUsers, db: Session = Depends(get_db)):
     db.refresh(new_user)
     return {"username": new_user.username, "email": new_user.email}
 
+
```

Comment:
Put request and response classes in a schemas.py file. So the application has 2 features. We'll go with a package by feature approach for this assignment. Try to structure the application into 2 packages auth and notes.